### PR TITLE
Component - inventory item

### PR DESCRIPTION
**Pull request by iva585**
- added header to the inventory page
- added list to the inventory page
- created inventory store
- created  update function for inventory items
- created delete function for inventory items
- updated story

### CODE REVIEW

File: src/app/components/InventoryItem/index.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.crossed-off {
```
Comment (Torfeltorfel) [inline src/app/components/InventoryItem/index.css:1]: Aren't you working with styled components?

File: src/app/components/InventoryItem/index.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      deleteInventoryItem({
+        ...props.item,
+      })
+    );
+  };
+  return (
+    <ListItem>
+      <ListItemIcon>
+        <Checkbox
+          edge="start"
```
Comment (Torfeltorfel) [inline src/app/components/InventoryItem/index.tsx:71]: naming `edge` is not clear here. What does it do?

**Review by Torfeltorfel — COMMENTED** [review]
Really good job, even though I have my difficulties to understand REDUX. Maybe you can take a look at the naming issue I found.

File: src/app/components/InventoryItem/index.css (hunk 1 above)
Comment (iva585) [reply src/app/components/InventoryItem/index.css:1]: No, I ditched them :D

File: src/app/components/InventoryItem/index.tsx (hunk 2 above)
Comment (iva585) [reply src/app/components/InventoryItem/index.tsx:71]: This isn't something I wrote, it is a MateriaI UI property.

**Pull request merged**